src/Lock.php

```diff
@@ -530,20 +632,27 @@ public static function showForItem(CommonDBTM $item)
         ) {
             $ipaddress->getFromDB($line['id']);
             if ($first) {
-                echo "<tr><th colspan='2'>" . IPAddress::getTypeName(Session::getPluralNumber()) . "</th></tr>\n";
+                echo "<tr>";
+                echo "<th width='10'></th>";
+                echo "<th>" . $ipaddress->getTypeName(Session::getPluralNumber()) . "</th>";
+                echo "<th>" . SoftwareVersion::getTypeName(Session::getPluralNumber()) . "</th>";
+                echo "<th></th>";
+                echo "<th>" . __('Automatic inventory') . "</th>";
+                echo "</tr>";
                 $first = false;
             }
 
             echo "<tr class='tab_bg_1'>";
-
             echo "<td class='center' width='10'>";
             if ($ipaddress->can($line['id'], UPDATE) || $ipaddress->can($line['id'], PURGE)) {
                 $header = true;
-                echo "<input type='checkbox' name='IPAddress[" . $line['id'] . "]'>";
+                echo "<input type='checkbox' name='Item_Disk[" . $line['id'] . "]'>";
```

Comment:
Are you sure of this change ?
```suggestion
                echo "<input type='checkbox' name='IPAddress[" . $line['id'] . "]'>";
```

Reply:
reverted to "IPAddress"